frontend/javascripts/viewer/model/actions/volumetracing_actions.ts

```diff
@@ -174,23 +174,27 @@ export const createCellAction = (activeCellId: number, largestSegmentId: number)
   } as const;
 };
 
-export const startEditingAction = (position: Vector3, planeId: OrthoView) =>
+export const startEditingAction = (positionInLayerSpace: Vector3, planeId: OrthoView) =>
   ({
     type: "START_EDITING",
-    position, // in layer space
+    positionInLayerSpace, // in layer space
```

Comment:
```suggestion
positionInLayerSpace,
```
The comment shouldn't be needed anymore due to the renaming of the variable